pkg/api/dashboard_public.go

```diff
@@ -90,7 +90,8 @@ func (hs *HTTPServer) QueryPublicDashboard(c *models.ReqContext) response.Respon
 		return handleDashboardErr(http.StatusInternalServerError, "Failed to get queries for public dashboard", err)
 	}
 
-	resp, err := hs.queryDataService.QueryDataMultipleSources(c.Req.Context(), nil, c.SkipCache, reqDTO, true)
+	user := models.SignedInUser{IsAnonymous: true, OrgId: c.OrgId}
```

Comment:
A bunch of things depend on a user with an orgId, so added that in for now. The `IsAnonymous` getting set to true is a crappy workaround for the POC so we can avoid the datasource guardian